cmd/linebot/main.go

```diff
@@ -77,8 +78,9 @@ func main() {
 	}
 
 	srv := &http.Server{
-		Handler: bot.handler,
-		Addr:    bot.config.Addr(),
+		Handler:           bot.handler,
+		Addr:              bot.config.Addr(),
+		ReadHeaderTimeout: readHeaderTimeout,
```

Comment:
fix
```
cmd/linebot/main.go:79:10: G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)
        srv := &http.Server{
                Handler: bot.handler,
                Addr:    bot.config.Addr(),
        }
```